bundle/edu.gemini.spdb.rollover.servlet/src/main/scala/edu/gemini/rollover/servlet/RolloverObservation.scala

```diff
@@ -48,9 +50,25 @@ object RolloverObservation {
       targetEnv  <- Option(dataObj.getTargetEnvironment)
       science    <- Option(targetEnv.getBase)
       name       <- Option(science.getTarget.getName)
-      ra         <- Option(science.getTarget.getRaDegrees)
-      dec        <- Option(science.getTarget.getDecDegrees)
-    } yield RolloverTarget(name, new Angle(ra, Angle.Unit.DEGREES), new Angle(dec, Angle.Unit.DEGREES))
+    } yield {
+
+      // Amazingly this is easier in Java
+      val when = o.getDataObject
```

Comment:
I wonder if this has any impact on ITAC, it calls the `RolloverServlet` to load targets
I'd think this is inconsequential for ITAC but I cannot say for sure